### ISSUE TITLE
Problems using SAPI4 with NVDA

### ISSUE THREAD

**Issue by nvaccessAuto** [nvaccess/nvda#1009]
**Reported by KevanGC on 2010-10-31 03:39**
I'm using eSpeak right now.
I have 2 SAPI4 engines installed. One of them is broke and doesn't work, but the other one does. When I select SAPI4 from the Synth dialog, NVDA goes to the broken engine by default, freezes and i have to restart NVDA to get it to work again. I was wondering if it might be possible to make an option allowing users to remove certain engines from the list of sapi4 voices. Maybe start out with no engines showing up in the list, and ask users which engines they want to ad. This option is already available with JAWS as far as I know.

**Comment by ehollig** [comment]
This does not sound like a bug in NVDA. Since SAPI4 is very old anyway and it is not a live disrupting issue, should this be closed?

**Comment by leonardder** [comment]
I agree. This could probably have been fixed by removing the broken engine, hacking te registry or even reinstalling the system.
